ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/ip_security.schema.yml

```diff
@@ -24,6 +24,31 @@ keys:
               description: |
                 Local IKE Identification.
                 Can be an IPv4 or an IPv6 address.
+            ike_lifetime:
+              type: int
+              convert_types:
+                - str
+              min: 1
+              max: 24
+              description: |
+                IKE lifetime in hours.
+            encryption:
+              type: str
+              valid_values:
+                - disabled
```

Comment:
so right now the options in CLI are:
```
host1(config-ipsec-ike)#encryption ?
  3des    Three key triple DES
  aes128  AES - 128-bit Advanced Encryption Standard
  aes256  AES - 256-bit Advanced Encryption Standard
  <cr>
host1(config-ipsec-ike)#show cli commands | grep encryption
[no|default] encryption [ aes128 | aes256 | 3des ]
```

and configuring `no encryption` (or `default encryption`) yields:

```
host1(config-ipsec-ike)#no encryption
host1(config-ipsec-ike)#exit
host1(config-ipsec)#show active
ip security
   ike policy BLAH
host1(config-ipsec)#show active all
ip security
   ike policy BLAH
      integrity sha256
      authentication pre-share
      ike-lifetime 8
      encryption aes128
```

so I would remove the `disabled` value as it does not really `disable` encryption contrary to the other case where `"null"` is an actual possibility

Reply:
What is encryption without any option? We may need to redesign the model to
```yaml
encryption:
  enabled: <bool>
  algorithm: <3des, aes128, aes256>
```
if there is some special meaning in `encryption` alone.

Reply:
```
host1(config-ipsec-ike)#encryption
host1(config-ipsec-ike)#show active
ip security
   ike policy BLEH
host1(config-ipsec-ike)#exit
host1(config-ipsec)#show active all
ip security
   ike policy BLEH
      integrity sha256
      authentication pre-share
      ike-lifetime 8
      encryption aes128
      dh-group 14
      version 2
      local-id ::
      remote-id ::
 ```
 
 no meaning on cEOS - still using default

Reply:
```
host1(config)#show run | se ike
ip security
   ike policy BLEH
host1(config)#
```

Reply:
OK, we can remove the `disabled` but otherwise keep the schema as is.